version.py

```diff
@@ -34,7 +34,7 @@
 # new level means in api_docs/changelog.md, as well as "**Changes**"
 # entries in the endpoint's documentation in `zulip.yaml`.
 
-API_FEATURE_LEVEL = 372  # Last bumped to interpret "(no topic)" as empty string.
+API_FEATURE_LEVEL = 376  # Last bumped for updating stream archive events.
```

Comment:
We can use the `tools/create-api-changelog` technique to avoid merge conflicts here.